city_scrapers/spiders/ind_indygo.py

```diff
@@ -17,79 +27,37 @@ def parse(self, response):
         Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
         needs.
         """
-        for item in response.css(".content-section:nth-child(4) ul li"):
+        meeting_section = response.css(".content-section:nth-child(4)")
+        section_title = meeting_section.css("h2::text").get()
+        meeting_year = section_title.split(" ")[0]
+        meeting_time = (
+            (response.css(".content-section:nth-child(4) p strong::text").get())
+            .split(":", 1)[1]
+            .split("EST")[0]
+            .strip()
+        )
+        print(meeting_time)
```

Comment:
Another stray `print` statement (see [this comment](https://github.com/City-Bureau/city-scrapers-indianapolis/pull/14/files#r1463711130)). I know print statements are great when you're trying to debug an issue or test that something is working, but please try to do your best to avoid including these in your commits. It keeps our git history cleaner and makes it easier for contributors and reviewers to see how changes in a file have evolved.